app/views/posts/index.html.erb

```diff
@@ -1 +1,8 @@
-<h1>Here is a list of posts for a given user</h1>
\ No newline at end of file
+<div id="postTemplate">
+   <%= render partial: "shared/single_user", locals: { user: @user } %>
+<% @user.posts.each do |post| %>
+     <%= render partial: "shared/single_post", locals: { post: post } %>   
```

Comment:
- [ ] Clicking on one of the posts should take you to the post's show page. Please kindly implement this.